redbot/core/events.py

```diff
@@ -55,7 +54,7 @@ async def on_ready():
         if bot._uptime is not None:
             return
 
-        bot._uptime = datetime.datetime.utcnow()
+        bot._uptime = datetime.now(timezone.utc)
```

Comment:
Bot's uptime from this var is exposed in `bot.uptime` property, so this change could break someone (+ it does break `[p]uptime` command). I don't think the change here is really needed.